include/unicorn/mips.h

```diff
@@ -61,7 +61,7 @@ typedef enum uc_cpu_mips64 {
 } uc_cpu_mips64;
 
 //> MIPS registers
-typedef enum UC_MIPS_REG {
+typedef enum uc_mips_reg {
```

Comment:
Can we avoid this renaming?

Reply:
We *can*, but I'm just really surprised this inconsistency was left here like that for a while - what if we keep the rename, but add an alias in the C side for compatibility to avoid breakage? (and then remove it way down the line in the future).